backend/controllers/controllers.go

```diff
@@ -0,0 +1,89 @@
+package controllers
+
+import (
+	"html/template"
+	"net/http"
+	"strconv"
+
+	"github.com/Aaryan376/to-do-list-go/services"
+)
+
+func IndexHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		http.NotFound(w, r)
```

Comment:
@Aaryan376 Can we address the issue no 4 in the same pr to change the error message

Reply:
Sure, will close this one and create a single PR for all the issues.